ThirdPartyAdapters/liftoffmonetize/liftoffmonetize/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java

```diff
@@ -310,28 +312,13 @@ public void loadRewardedAd(
 
     Context context = mediationRewardedAdConfiguration.getContext();
 
-    VungleInitializer.getInstance()
-        .initialize(
-            appID,
-            context,
-            new VungleInitializationListener() {
-              @Override
-              public void onInitializeSuccess() {
-                rewardedAd = vungleFactory.createRewardedAd(context, placement, adConfig);
-                rewardedAd.setAdListener(VungleMediationAdapter.this);
-                if (!TextUtils.isEmpty(userId)) {
-                  rewardedAd.setUserId(userId);
-                }
-
-                rewardedAd.load(null);
-              }
-
-              @Override
-              public void onInitializeError(AdError error) {
-                Log.w(TAG, error.toString());
-                VungleMediationAdapter.this.mediationAdLoadCallback.onFailure(error);
-              }
-            });
+    rewardedAd = vungleFactory.createRewardedAd(context, placement, adConfig);
```

Comment:
The VungleFactory should be removed... It's a wrapper class that just returns a new instance... This should be `RewardedAd(context, placement, adConfig)`

Reply:
+1

Reply:
We can't. Because it is added from google side for unit tests.